src/app/common/devdependencies.go

```diff
@@ -0,0 +1,29 @@
+package common
+
+import (
+	"fmt"
+	"os"
+
+	"gopkg.in/yaml.v3"
+)
+
+// ExcludedDependencies represents the structure of the YAML file.
+type ExcludedDependencies struct {
```

Comment:
NIT: is it exported due to yaml Unmarshal? Aren't just the fields forced to be exported?